webui/tests/cypress/support/commands.js

```diff
@@ -51,7 +51,7 @@ Cypress.Commands.add('login', credentials => {
     password: saltAndHashPassword(DEFAULT_TEST_PASSWORD),
     username: DEFAULT_TEST_USERNAME,
   };
-  cy.request('POST', '/login', credentials)
+  cy.request('POST', '/api/v1/auth/login', credentials)
```

Comment:
nit: we'd probably won't have to construct these URLs in many places but if we do the prefix could be shared.

Reply:
thoughts on using `cypress.env.json` for the purpose of these shared info?

Reply:
hmm if we think of these as values people might want to change that seems like a good place otherwise hardcoded as a const somewhere IMO works as well.

Reply:
👍  will extract into `constants.ts` and put shared constants there